src/amo/components/AddonReviewListItem/index.js

```diff
@@ -311,6 +316,7 @@ export function mapStateToProps(
   return {
     editingReview,
     isAuthenticated: isAuthenticated(state),
+    isAdmin: isAdmin(state),
```

Comment:
I would prefer something like `siteUserIsAdmin` to match the `siteUser` prop. Also, this clarifies which user it refers to (this component already deals with add-on owner users and review author users).

Reply:
I also changed the prop for `isAuthenticated`.